python/lsst/ip/diffim/makeKernelBasisList.py

```diff
@@ -31,30 +31,37 @@
 
 def makeKernelBasisList(config, targetFwhmPix=None, referenceFwhmPix=None,
                         basisDegGauss=None, metadata=None):
-    """Generate the appropriate Kernel basis based on the Config
+    """Generate the delta function or Alard-Lupton kernel bases depending on the Config.
+    Wrapper to call either `lsst.ip.diffim.makeDeltaFunctionBasisList` or
+    `lsst.ip.diffim.generateAlardLuptonBasisList`.
 
     Parameters
     ----------
-    config : TODO: DM-17458
-        TODO: DM-17458
-    targetFwhmPix : TODO: DM-17458, optional
-        TODO: DM-17458
-    referenceFwhmPix : TODO: DM-17458, optional
-        TODO: DM-17458
-    basisDegGauss : TODO: DM-17458, optional
-        TODO: DM-17458
-    metadata : TODO: DM-17458, optional
-        TODO: DM-17458
+    config : `lsst.ip.diffim.PsfMatchConfigAL`
+        Configuration object.
+    targetFwhmPix : `float`, optional
+        Passed on to `lsst.ip.diffim.generateAlardLuptonBasisList`. Not used otherwise.
+    referenceFwhmPix : `float`, optional
+        Passed on to `lsst.ip.diffim.generateAlardLuptonBasisList`. Not used otherwise.
+    basisDegGauss : list of `int`, optional
+        Passed on to `lsst.ip.diffim.generateAlardLuptonBasisList`. Not used otherwise.
+    metadata : `lsst.daf.base.PropertySet`, optional
+        Passed on to `lsst.ip.diffim.generateAlardLuptonBasisList`. Not used otherwise.
```

Comment:
For each of these, perhaps say "Not used for delta function basis sets" instead of "Not used otherwise".,